### DISCUSSION TITLE
GetComponent로 제페토스크립트를 얻으려고 하면 null이 반환되는 이슈

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#537] [Issue Report (Legacy)]
ZepetoScript A가 있을 때
모바일에서 ```GetComponent<A>``` 혹은 ```AddComponent<A>``` 를 했을 때 항상 null이 반환됩니다.
PC에서는 정상 동작합니다.

이때 해결방법은 새로운 ZepetoScript B를 만들어서 A스크립트의 코드내용을 붙여넣기하고, A대신 B스크립트를 사용하는 것입니다.

재현방법은 모르겠으며 간헐적으로 랜덤하게, 지속적으로 발생합니다. (한번 발생하면 그 스크립트에 대해서는 코드 내용과 관계없이 항상 발생)
월드개발 진행에 큰 차질을 주는 이슈라 중요하게 봐주셨으면 좋겠습니다.

**Comment by Cha-SeongHun** [comment]
정정합니다. 새 제페토스크립트 B를 만들어 사용해도 확률적으로 해결이 될 때가 있고, 동일하게 null이 반환되며 해결이 되지 않을 때가 있습니다.
 또한 어떤 때는 PC에서도 재현되기도 합니다 (재현방법은 잘 모르겠습니다)

**Reply by x4cf** [reply]
사용중인 OS와 제페토 API버전을 알 수 있을까요?

**Reply by Cha-SeongHun** [reply]
OS는 Windows 11 및 안드로이드 12(갤럭시 Z 폴드 2), 제페토 API 버전은 1.15(현재 최신 버전)입니다.
